src/ReactWrapper.jsx

```diff
@@ -175,6 +177,8 @@ class ReactWrapper {
     if (this.root !== this) {
       throw new Error('ReactWrapper::instance() can only be called on the root');
     }
+    if (!this.component.state.mount) return instanceMap.get(this.component);
```

Comment:
That seems like a reasonable companion change anyways; `getInstance` shouldn't throw if it doesn't have an instance, it should just not return one.